README.md

```diff
@@ -1,2 +1,239 @@
-# python-bindings
-Python language bindings for preCICE :construction: work in progress :construction:
+Python language bindings for the C++ library preCICE
+----------------------------------------------------
+
+<a style="text-decoration: none" href="https://travis-ci.org/precice/python-bindings" target="_blank">
+    <img src="https://travis-ci.org/precice/python-bindings.svg?branch=develop" alt="Build status">
+</a>
+
+This package provides python language bindings for the C++ library [preCICE](https://github.com/precice/precice). Note that the first three digits of the version number of the bindings indicate the preCICE version that the bindings support. The last digit represents the version of the bindings. Example: `v2.0.0.1` of the bindings represents version `1` of the bindings which is compatible with preCICE `v2.0.0`.
+
+# Required dependencies
+
+* **preCICE**: Refer to (the preCICE wiki)[https://github.com/precice/precice/wiki#1-get-precice] for information on installation.
+
+# Installing the package
+
+We recommend [using pip3](https://github.com/precice/precice/blob/develop/src/precice/bindings/python/README.md#using-pip3) for the sake of simplicity.
+
+## Using pip3
+
+### preCICE system installs
+
+For system installs of preCICE, this works out of the box.
+
+You can either install from PyPI:
```

Comment:
What about using pip to download and install from the github repo as an archive?

Reply:
Done e440e7c